doc/howtos/booting-and-installation.rst

```diff
@@ -147,6 +147,126 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
 #. Plug in the second micro-USB cable to the J16 connector as instructed by the running flashall script.
 #. Wait for all the images to flash. You will see the progress on the flasher.
 #. Once flashing is done, the image will automatically boot up and auto-login as ``root``, no password is required.
+   
+BeagleBone Black
+================
+
+BeagleBone Black is booted from a microSD card with MBR (Master Boot Record) and not GPT (GUID Partition Table) partitions.
+Most freshly unpackaged microSD cards come with MBR partitions, but previously used ones might not.
+
+You'll probably need an adapter to use the microSD card on your host computer. If you use a microSD-to-SD adapter, 
+it will likely show up as /dev/mmcblk0. You can verify the device name assigned by using ``dmesg`` or the
+``lsblk`` command to look for the device name for the microSD card (check for a device with the size you're expecting). 
+
+In our examples below, we're using an 8GB microSD card in an SD adapter that's showing up as /dev/mmcblk0  
+(numbers and device name maybe different for your device and system).
+
+You should download the ``ostro-image-dev-beaglebone*.rootfs.tar.bz2`` image from the `Ostro Project download server`_
```

Comment:
I think we should turn that into a complete list of files that need to be downloaded:
- rootfs (there are different rootfs images available, <insert pointer to image variants in our documentation> and let the user choose the one he wants (pick one yourself for the rest of the exercise and mention that this is an example).
- dtb file
- u-boot.img file

Multiple versions of these exist in the same directory, we should add some verbiage to indicate that they are essentially the same (and I suppose they're in fact symlinks, @kad?)

@kad, another thing I've noticed is that there is a zImage-am335x-bone.dtb file published which I'm assuming is for the original BeagleBone board (vs. the newer Black one). Can we remove this from the list of published files given we don't support the BeagleBone?